exercises/hamming/example.js

```diff
@@ -1,24 +1,12 @@
-class Hamming {
-
-  compute(strand1, strand2) {
-    const len1 = strand1.length;
-    const len2 = strand2.length;
-    if (len1 !== len2) {
-      throw new Error('left and right strands must be of equal length');
-    }
-
-    let distance = 0;
-    let idx = -1;
-    const end = len1; // there could be len2, they're equal
-    while (++idx < end) {
-      if (strand1[idx] !== strand2[idx]) {
-        distance++;
-      }
-    }
-
-    return distance;
+export const compute = (strand1, strand2) => {
+  const len1 = strand1.length;
+  const len2 = strand2.length;
+  if (len1 !== len2) {
+    throw new Error('left and right strands must be of equal length');
   }
 
-}
-
-export default Hamming;
+  return [...strand1].reduce(
```

Comment:
Since we know the strands must be of equal length, this could also be written, perhaps more semantically, like so:
```suggestion
  return [...strand1].filter((el, idx) => el !== strand2[idx]).length;
```